lib/mix/lib/mix/task.ex

```diff
@@ -275,6 +277,13 @@ defmodule Mix.Task do
     end
   end
 
+  defp output_task_debug_info(task, args, proj) do
+    proj_str = if is_nil(proj), do: "", else: " (inside #{inspect proj})"
```

Comment:
I usually stick with conditions that return booleans when possible but I agree

```elixir
if proj, do: " (inside #{inspect proj})", else: ""
```

read better.